graspologic/layouts/auto.py

```diff
@@ -249,6 +245,27 @@ def _node2vec_for_layout(
     return graph, tensors, labels
 
 
+def _to_undirected(graph: nx.DiGraph) -> nx.Graph:
+    sym_g = nx.Graph()
+    weighted = nx.is_weighted(graph)
+    for source, target, weight in graph.edges.data("weight"):
```

Comment:
doesn't this line break for an unweighted graph?

Reply:
nope, weight just becomes None and it runs fine. Is there a better/safer way of iterating through edges of an unweighted graph?